pkg/nsx/services/inventory/inventory.go

```diff
@@ -334,3 +334,23 @@ func (s *InventoryService) updateInventoryStore() error {
 func (s *InventoryService) UpdatePendingAdd(externalId string, inventoryObject interface{}) {
 	s.pendingAdd[externalId] = inventoryObject
 }
+
+func (s *InventoryService) Cleanup(ctx context.Context) error {
+	errList := []interface{}{}
```

Comment:
var errList []interface{}
is recommended

Reply:
it's the same since not check if errList is nil